src/lcm/distributions.py

```diff
@@ -3,15 +3,15 @@
 
 
 def lognormal(sd, size):
-    return np.random.lognormal(0, sd, size)
+    return np.random.lognormal(0, sd, size)  # noqa: NPY002
```

Comment:
Pls open an issue to search & remove NPY002 https://numpy.org/doc/stable/reference/random/legacy.html#legacy

Will stop working at some point, so better make the move soon.

Reply:
Done in #6